README.md

```diff
@@ -0,0 +1,215 @@
+# Docker image for Petals Deployer
+
+This image allows to configure and run a Petals CLI instance to deploy artifacts onto a running Petals ESB instance.
+
+Connection parameters and artifact list must be set at launch.
+Artifacts can be deployed by 
+* mounting a docker volume containnig artifacts zipfiles and giving files to deploy.
+* hosting artifacts on a nexus and giving artifacts url.
+>Both method can be used at once but artifacts as parameters will always be deployed first.
+
+
+## Launch a container from this image
+
+```properties
+# Download the image
+docker pull petalslink/petals-deployer:latest
+
+# Start in detached mode (with default config)
+docker run -d -p 80:80 --name petals-deployer petalslink/petals-deployer:latest
+
+# Start example #1
+docker run \
+	-d -p 80:80 --name petals-deployer \
+	-v /home/user/Petals/artifacts/:/tmp/artifact/ \
+	-e "ENV_DEPLOY_FILE=/tmp/artifact/filesToDeploy.txt" \
+	-e "ENV_CLI_HOST=172.17.0.2" \
+	petalslink/petals-deployer:latest
+
+# Start example #2
+docker run \
+	-d -p 80:80 --name petals-deployer \
+	-e "ENV_CLI_HOST=172.17.0.5" \
+	-e "ENV_DEPLOY_URLS=mvn:org.ow2.petals/petals-bc-soap/LATEST/zip;mvn:org.ow2.petals/petals-bc-ftp/LATEST/zip" \
+	petalslink/petals-deployer:5.2.0
+
+# Verify the container was launched
+docker ps
+
+# Watch what happens/happened
+docker logs petals-deployer
+
+# Verify the ports used on the host (example)
+sudo lsof -i :80
+
+# Get information about the container
+docker inspect petals-deployer
+
+# Introspect the container
+docker exec -ti petals-deployer /bin/bash
+```
+
+The example shows how to get the last version.
+You can obviously change the version. Each Petals container release has a corresponding deployer release, with a matching version. As an example, to get a deployer for **Petals ESB 5.1.0** (which would actually be **Petals CLI 3.0.0**), just type in `docker pull petalslink/petals-deployer:5.1.0`.
+> You can choose a specific **Petals CLI** version by building your own image
+
+## Run parameters
+
+When running this deployer, you must specify it which artifacts to deploy on which Petals ESB container instance.
+> Please note that service assemblies require components which may require shared libraries. Be sure to deploy in the correct order. Every artifact will be started (if possible) after being deployed.     
+
+You must use the following parameters :
+
+| Argument | Optional | Default | Description |
+| -------- | :------: | :-----: | ----------- |
+| CLI_PETALS_HOST | yes | `localhost` | The petals container hostname. |
```

Comment:
`localhost` will never work.
With Docker, it is better to set the bridge address: `172.17.0.1` (type in `ifconfig`, you will see this is the host address for the `docker0` network interface).